homework/content.js

```diff
@@ -1,37 +1,43 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function(exports) {
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
+  var NoteContentManager = function() {
+    this._wrapper = document.querySelector('#note-content-wrapper');
   }
+  
+  NoteContentManager.prototype = {
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
+    start() {
+      window.addEventListener('note-open', (function(event) {
```

Comment:
Instead of binding an anonymous function, you should set the `listener` argument as `this`, and deal with the event in the `handleEvent` method.